tox.ini

```diff
@@ -1,13 +1,23 @@
 [tox]
-envlist = pep8
+envlist = pep8,py27
```

Comment:
we're testing Python2 only until the Python3 `SyntaxError` mentioned in #51, #54 and #55 gets fixed